gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -1538,8 +1538,8 @@ function GM:TTTCheckForWin()
 	elseif b == 1 then -- just 1 team is alive
 		return alive[1]
 	else -- rare case: nobody is alive, e.g. because of an explosion
-		--return WIN_NONE -- bees_win
-		return WIN_TRAITOR
+		--return WIN_NONE -- none_win
+		return WIN_TRAITOR -- TODO WIN_NONE?
```

Comment:
i think keeping this as an traitor win is fine... well as long as we expect people to not play without traitors =D

i think we should decide this now

Reply:
I think this should be win_none or win_tie or something like this

Reply:
Inconsistency will lead to misunderstanding, so I tend using WIN_NONE as well. Or we use an Option, so Server-Owner can decide on their own?

Reply:
No, please don't add yet another convar. I just thought about a distinction between those two, so there can be different round end texts. Like "Time is up", "No one won" and "It's a tie"
But now that I write it down, the last two are probably the same anyway. So I'd pledge for using `WIN_NONE`